tests/GenerateFormSubmitErrors.php

```diff
@@ -0,0 +1,28 @@
+<?php
+use PHPUnit\Framework\TestCase;
+
+require_once 'index.php';
+
+class DisplayAllRecordsTest extends TestCase
```

Comment:
You should add an additonal test here that provides all fields as invalid to make sure you get all the expected errors. This assertion might help https://www.geeksforgeeks.org/phpunit-assertarrayhaskey-function/